concent_api/core/payments/sci_backend.py

```diff
@@ -0,0 +1,140 @@
+import math
```

Comment:
So general issue here is that you are mixing functions which use API with those which don't. For example, `sum_payments` is not using API at all. Please note that this function is almost identical with the one in mock backend (the assert should be there too). If for example in there future we will create a database backend, this function will be actually the same there. 

So, please use here in backend only functions which really use something from outside of the Concent. And the functions which don't should be extracted into single place. At this moment it should be fine to put them into `message_handlers.py`. Just to clarify, those are:
* `sum_payments`
* `sum_subtask_price`
* `sum_amount_price_for_provider`
* `get_clients_eth_accounts`

You can still mock those in tests if you need. Please also remember to remove them from mock backend and base.

Reply:
Done